examples/medplum-chat-demo/src/utils.tsx

```diff
@@ -0,0 +1,186 @@
+import { Filter, getReferenceString, Operator, parseReference, SearchRequest } from '@medplum/core';
+import {
+  Communication,
+  Encounter,
+  EncounterParticipant,
+  Practitioner,
+  QuestionnaireResponse,
+  QuestionnaireResponseItemAnswer,
+  Resource,
+} from '@medplum/fhirtypes';
+
+export function cleanResource(resource: Resource): Resource {
```

Comment:
I think this function isn't used in this demo, and should be removed

Reply:
This is being used in PatientDetails and ResourcePage